entity/contents/filter_contents.h

```diff
@@ -91,4 +106,37 @@ class BlendFilterContents : public FilterContents {
   FML_DISALLOW_COPY_AND_ASSIGN(BlendFilterContents);
 };
 
+/*******************************************************************************
+ ******* GaussianBlur1DFilterContents
+ ******************************************************************************/
+
+class GaussianBlur1DFilterContents final : public FilterContents {
+ public:
+  GaussianBlur1DFilterContents();
+
+  ~GaussianBlur1DFilterContents() override;
+
+  void SetRadius(Scalar radius);
+
+  void SetDirection(Point direction);
```

Comment:
Should this take `Vector direction` as an argument?

Reply:
Do we have a Vector class somewhere? Do you mean Vector3? I think more dimensions would just act as dampeners.